### ISSUE TITLE
OSQP and L-BFGS

### ISSUE THREAD

**Issue by damienhocking** [oxfordcontrol/osqp#306]
Has anyone worked on using a limited-memory BFGS with OSQP instead of a full Hessian?

**Comment by goulart-paul** [comment]
As far as I am aware, no.

**Comment by damienhocking** [comment]
We might have a use for that and could do the development. Do you take external contributions?

**Comment by goulart-paul** [comment]
Yes, we are very happy to take external contributions from anyone who signs up the project's open source license agreement.

Note that you might want to develop something like that against the v1.0 dev version, since we changed the interface somewhere there to allow greater flexibility for users to interface alternative solvers, different BLAS type implementations etc.

**Comment by damienhocking** [comment]
OK, I'll take a look at that branch. I have a L-BFGS written in C++, it wouldn't be hard to port that to C. I'm happy to put that under your license. My only constraint is the time I have to work on it.

**Comment by damienhocking** [comment]
The last time that v1.0 dev was updated was 2019. You're sure that's the right one?

**Comment by goulart-paul** [comment]
Sadly, yes.   It is still live and we will release it eventually, but events over the last year have rather derailed our schedule.   I hope to make some progress to deployment over the summer.
